### DISCUSSION TITLE
React Frontend Q&A

### DISCUSSION THREAD

**Discussion by nishuranjan** [nishuranjan/learningdharaportal#24] [General]
Please keep asking questions here

**Comment by SHREYASH96OG** [comment]
What is the purpose of render() in React.

**Reply by nishuranjan** [reply]
React renders HTML to the web page by using a function called render(). 

The purpose of the function is to display the specified HTML code inside the specified HTML element. 

In the render() method, we can read props and state and return our JSX code to the root component of our app.

**Reply by nishuranjan** [reply]
Please have a look on official website as well - https://legacy.reactjs.org/docs/rendering-elements.html

**Reply by nishuranjan** [reply]
For quick check try here - https://codepen.io/gaearon/pen/edyBaE?editors=1010

**Reply by nishuranjan** [reply]
Updating the Rendered Element

https://codepen.io/gaearon/pen/gwoJeZ?editors=1010
